package.json

```diff
@@ -165,7 +165,7 @@
     "instrument": false
   },
   "engines": {
-    "node": ">=6.0.0"
+    "node": ">=9.0.0"
```

Comment:
But this is 9 not 8 ><

Reply:
What's the point? I have rollback in another commit.

Reply:
Got it. soz